src/OpenSSL/crypto.py

```diff
@@ -1452,6 +1477,46 @@ def add_cert(self, cert):
         if not result:
             _raise_current_error()
```

Comment:
since you’re touching the code around here you could `_openssl_assert`-ize this as well :)